modules/apps/image/image-service/src/main/java/com/liferay/image/service/internal/upgrade/v1_0_0/ImageStorageUpgradeProcess.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.image.service.internal.upgrade.v1_0_0;
+
+import com.liferay.document.library.kernel.store.DLStoreUtil;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.dao.orm.ActionableDynamicQuery;
+import com.liferay.portal.kernel.dao.orm.RestrictionsFactoryUtil;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.model.Image;
+import com.liferay.portal.kernel.service.ImageLocalService;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.io.InputStream;
+
+/**
+ * @author Adolfo Pérez
+ */
+public class ImageStorageUpgradeProcess extends UpgradeProcess {
+
+	public ImageStorageUpgradeProcess(ImageLocalService imageLocalService) {
+		_imageLocalService = imageLocalService;
+	}
+
+	@Override
+	protected void doUpgrade() throws PortalException {
+		ActionableDynamicQuery actionableDynamicQuery =
+			_imageLocalService.getActionableDynamicQuery();
+
+		actionableDynamicQuery.setAddCriteriaMethod(
+			dynamicQuery -> dynamicQuery.add(
+				RestrictionsFactoryUtil.eq("companyId", 0L)));
```

Comment:
@adolfopa forgive me if I'm missing something, but I think you'll want to remove this restriction. The entries in the image table _do_ have a valid companyId, and it's even being used on line 56. I think acting on all images is what we want, since in theory every singe file written by the DLHook through Image service was written to the "0" folder.

Reply:
Good catch. I've pushforced the changes; tested it locally and the company logo was migrated correctly.

![company-logo-upgrade](https://user-images.githubusercontent.com/729897/117677842-1c78bd00-b1af-11eb-84c1-216fdac094cd.png)